software/controller/lib/debug/eeprom_cmd.cpp

```diff
@@ -27,10 +27,10 @@ ErrorCode EepromHandler::Process(Context *context) {
 
   // Process subcommand
   switch (context->request[0]) {
```

Comment:
think it would be cleaner to do something like

  auto subcmd = static_cast<Subcommand>(context->request[0]);
  switch (subcmd) {

and avoid the casts in the switch statements.